arangosh/Import/ImportHelper.h

```diff
@@ -28,6 +28,7 @@
 #include <atomic>
 
 #include "AutoTuneThread.h"
+#include "quick_hist.h"
```

Comment:
File name convention here differs from all other file names used in ArangoDB and that are our own code.